core/src/main/java/com/linecorp/armeria/internal/server/DefaultServiceRequestContext.java

```diff
@@ -188,6 +189,8 @@ public DefaultServiceRequestContext(
         maxRequestLength = cfg.maxRequestLength();
         this.additionalResponseHeaders = additionalResponseHeaders;
         this.additionalResponseTrailers = additionalResponseTrailers;
+
+        shouldReportUnloggedException = true;
```

Comment:
What do you think of setting the default value in the field declaration?
It might be easier to know the default value of `shouldReportUnloggedException`.
```java
private boolean shouldReportUnloggedException = true;
```

Reply:
I think that will be more clear for the users. Thanks !